### DISCUSSION TITLE
Diamond v5 - landing.css inconsistent URLs

### DISCUSSION THREAD

**Discussion by vitsin** [primefaces/community#2587] [PrimeFaces Templates]
In the same diamond-layout/css/landing.css there are URLs like these:
```
url("../fonts/nunito-v12-latin-ext_latin-600.woff")
url("../icons/primeicons.eot")
etc
```

In the exact same landing.css there are 2 URLs (that I suspect are bugs)
that are like these:
```
1) background-image:url(/diamond/javax.faces.resource/images/pages/landing/bg-header.svg.xhtml?ln=diamond-layout)
and 
2) background-image:url(/diamond/javax.faces.resource/images/pages/landing/bg-contact.svg.xhtml?ln=diamond-layout)
```

So when I use landing.html which is referencing:
```
<link rel="stylesheet" href="resources/diamond-layout/css/landing.css"/>
```
I'm getting HTTP 404 on bg-header.svg.xhtml and bg-contact.svg.xhtml

But when I modify above URLs to these:
```
url(../images/pages/landing/bg-header.svg)
url(../images/pages/landing/bg-contact.svg)
```

all works fine, images show fine.

**Comment by tandraschko** [comment]
will fix it for the upcoming 15.0.1 release
